CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `quint repl` is printing the same version number as returned by `quint --version` (#804)
 - add the command `.seed` in REPL (#812)
 - fix `quint run` to output compile errors again (#812)
+- fix `import from` in REPL (#834)
```

Comment:
I think We don’t need a release not for this because the break wasn’t in a release?